ingestion/src/metadata/ingestion/source/database/mssql/utils.py

```diff
@@ -488,3 +491,14 @@ def get_view_names(
     )
     view_names = [r[0] for r in connection.execute(query_)]
     return view_names
+
+
+def get_sqlalchemy_engine_dateformat(engine: Engine) -> Optional[str]:
+    """
+    returns sqlaclhemdy engine date format by running config query
+    """
+    result = engine.execute(GET_DB_CONFIGS)
+    for row in result:
+        if row["Set Option"] == "dateformat":
```

Comment:
lets make it  `row.get("Set Option")` to avoid any unexpected key errors 

same for `row.get("Value")`